cfspopcon/formulas/scrape_off_layer/two_point_model/separatrix_SOL_collisionality.py

```diff
@@ -0,0 +1,27 @@
+"""Routine to calculate the separatrix SOL collisionality."""
+
+from ....unit_handling import Unitfull, ureg, wraps_ufunc
+
+
+@wraps_ufunc(
+    return_units=dict(separatrix_SOL_collisionality=ureg.dimensionless),
+    input_units=dict(separatrix_electron_density=ureg.m**-3, separatrix_electron_temp=ureg.eV, parallel_connection_length=ureg.m),
+)
+def calc_separatrix_SOL_collisionality(
+    separatrix_electron_density: Unitfull,
+    separatrix_electron_temp: Unitfull,
+    parallel_connection_length: Unitfull,
+) -> Unitfull:
+    """Calculate the separatrix SOL collisionality.
+
+    #TODO: needs reference
```

Comment:
@mwigram

Reply:
Reference: "The Plasma Boundary of Magnetic Fusion Devices", Peter C Stangeby, Chapter 4